Source/WebGPU/WebGPU/CommandEncoder.swift

```diff
@@ -23,6 +23,11 @@
  * OF THIS SOFTWARE, EVEN IF ADVISED OF THE POSSIBILITY OF SUCH DAMAGE.
  */
 
+import Metal
+
+private let WGPU_COPY_STRIDE_UNDEFINED = 0xffffffff
+private let MTLBlitOptionNone: UInt = 0
```

Comment:
is there a swift equivalent? In Swift, the docs say MTLBlitOption is a struct - https://developer.apple.com/documentation/metal/mtlblitoption

Reply:
There isn't.It might be that MTLBiltOptionsNone is the default value but I just wanted to be sure.

Reply:
Can we import MTLBlitOptionNone from Metal instead of hard-coding it ourselves?

Reply:
It's not exposed in Swift. @geoffreygaren.
https://developer.apple.com/documentation/metal/mtlblitoption/mtlblitoptionnone